diff -urN bloboats-1.0.2/Makefile bloboats-1.0.2-patched/Makefile
--- bloboats-1.0.2/Makefile	2010-11-24 05:27:16.000000000 +1000
+++ bloboats-1.0.2-patched/Makefile	2014-02-13 01:34:10.169531991 +1100
@@ -90,7 +90,7 @@
 BINARY = $(BINARY_$(TARGET))
 
 ### CXX-Flags
-CXXFLAGS_DEFAULT = -g `$(SDLCONFIG) --cflags` -o bin/$(BINARY) -O2
+CXXFLAGS_DEFAULT = -g `$(SDLCONFIG) --cflags` -o bin/$(BINARY) $(CFLAGS)
 CXXFLAGS_FREEBSD = -g `$(SDLCONFIG) --cflags` -I/usr/X11R6/include -o bin/$(BINARY) -O2
 CXXFLAGS_NETBSD = -g `$(SDLCONFIG) --cflags` -O2 -I/usr/pkg/include
 CXXFLAGS_MACOSX = -g `$(SDLCONFIG) --cflags` -o bin/$(BINARY) -O2
